code/modules/food_and_drink/drinks.dm

```diff
@@ -329,8 +332,12 @@
 	attack_self(mob/user as mob)
 		var/drop_this_shit = 0 //i promise this is useful
 		if (src.is_sealed)
+			IN
 			is_sealed = 0
 			can_chug = 1
+			splash_all_contents = 1
+			incompatible_with_chem_dispensers = 0
```

Comment:
Also should be done for `is_sealed` and `can_chug` on the two lines above probably.
```suggestion
			splash_all_contents = TRUE
			incompatible_with_chem_dispensers = FALSE
```